clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileSystem.java

```diff
@@ -214,6 +216,7 @@ public FSDataInputStream open(Path path, int bufSize) throws IOException {
     public FSDataOutputStream create(Path path, FsPermission permission, boolean overwrite,
                                      int bufferSize, short replication, long blockSize,
                                      Progressable progress) throws IOException {
+        LOG.trace("creat_op: {}", path);
```

Comment:
Is "creat" intentional?  I know the origin story of the `creat` system call, but it might be easier for users to search their logs for "create" -- particularly given that Hadoop call it "create".

Reply:
it's a typo :) thanks